unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Scripts/ControlsModule/SliderSettingsControlView.cs

```diff
@@ -15,23 +15,21 @@ public class SliderSettingsControlView : SettingsControlView
         [SerializeField] private Slider slider;
         [SerializeField] private TextMeshProUGUI indicatorLabel;
 
-        public Slider sliderControl => slider;
-
         private SliderControlModel sliderControlConfig;
         private SliderSettingsControlController sliderController;
 
-        public override void Initialize(SettingsControlModel controlConfig, SettingsControlController settingsControlController)
+        public override void Initialize(SettingsControlModel model, SettingsControlController controller)
```

Comment:
same problem, view->controller coupling

Reply:
I am fully agree with you (also saw this strange implementation of the MVC, when last time touched Settings UI architecture). However, in this PR I changed only naming's of parameters. I checked now how much work it would be to split correctly Model, View and Controller and I think it is quite a big refactoring, that should be taken as a separate issue (I will create a task for it).

Analysis shows:
- `SettingsControlView `has 4 inheritors. Operates directly with both `Model `(only read data) and `Controller `(call method from it to change the Model and Rise view related events).
- `SettingsControlController `has 37 inheritors. Changes `Model`, used by `View` to get/set data from/to the `Model`, and also rise events when `View` changed (via call by the `View` itself)
- `SettingsControlModel `has 3 inheritors. It is a simple Data-Transfer-Object/Config that doesn't have any business logic inside.

Coupling is handled hierarchically. Entry point is call from Kernel 
`HUDBridge.ConfigureHUDElement()` -> `HUDController.ConfigureHUDElement(SETTINGS_PANEL)` -> `SettingsPanelHUDController.Initialize()` -> ( `SettingsPanelHUDView.Initialize()` -> `CreateSections()` ) -> `SettingsPanelHUDController.AddSection()` -> ( `SettingsSectionView.Initialize()` -> `CreateWidgets()` ) -> `SettingsSectionController.AddWidget()` -> ( `SettingsWidgetView..Initialize()` -> `CreateControls()` ) -> `SettingsWidgetController..AddControl()` -> `SettingsControlView.Inizialize()`

Thus, wrong implementation should be refactored on 4 levels  -  `SettingsPanelHUD ` -> `Section` - > `Widget` -> `Control`

Reply:
great analysis, thanks!